akka-cluster/src/test/scala/akka/cluster/sbr/SplitBrainResolverSpec.scala

```diff
@@ -846,15 +846,15 @@ class SplitBrainResolverSpec
       val decision1 = strategy1.decide()
       decision1 should ===(AcquireLeaseAndDownUnreachable(Duration.Zero))
       strategy1.nodesToDown(decision1) should ===(side2Nodes)
-      val reverseDecision1 = strategy1.reverseDecision(decision1)
+      val reverseDecision1 = strategy1.reverseDecision(decision1.asInstanceOf[AcquireLeaseDecision])
```

Comment:
Annoying to have to add the case, but at least the prod code is well-typed now and we force the tests to follow the types.